custom_components/meural/pymeural.py

```diff
@@ -43,19 +51,31 @@ async def request(self, method, path, data=None) -> Dict:
             else:
                 kwargs["json"] = data
         with async_timeout.timeout(10):
-            resp = await self.session.request(
-                method,
-                url,
-                headers={
-                    "Authorization": f"Token {self.token}",
-                    "x-meural-api-version": "3",
-                },
-                raise_for_status=True,
-                **kwargs,
-            )
+            try:
+                resp = await self.session.request(
+                    method,
+                    url,
+                    headers={
+                        "Authorization": f"Token {self.token}",
+                        "x-meural-api-version": "3",
+                    },
+                    raise_for_status=True,
+                    **kwargs,
+                )
+            except ClientResponseError as err:
+                _LOGGER.warning('Meural: Sending Request failed. Re-Authenticating.')
+                self.token = None
+                return
+            except Exception as err:
+                _LOGGER.warning('Meural: Sending Request failed. Ignoring: %s' %err)
```

Comment:
This happens when the device is powered off. So ignoring without renewing the token.